src/public/utils.js

```diff
@@ -159,6 +162,23 @@ export function utils_set_sections_history(state) {
     }
 }
 
+export async function utils_load_history(_is_resumed) {
+    $w("#repeaterHist").onItemReady( ($item, itemData, index) => {
+        utils_config_items($item, g_hist_map, itemData);
+        utils_set_sections_history(SECTION_STATE_DATA);
+    });
+    
+    let history = (await be_mod_utils_get_history()).reverse();
+
+	if(_is_resumed)
+    	history = history.length >= QTDE_ITENS_RESUMO ? history.slice(0 , QTDE_ITENS_RESUMO) : history; // limits items on resume
```

Comment:
I would slice in the backend, `be_mod_utils_get_history` can receive a parameter, specially because you can add a query to just return you X items

Reply:
Commented in another comment.